tutorials/llm/llama-3/README.rst

```diff
@@ -40,8 +38,16 @@ This notebook shows how to perform LoRA PEFT on **Llama 3 8B Instruct** using `P
 
 To get started
 ^^^^^^^^^^^^^^
+1. Download the `Llama 3 8B Instruct .nemo <https://catalog.ngc.nvidia.com/orgs/nvidia/teams/nemo/models/llama-3-8b-instruct-nemo>`__ from NVIDIA NGC using the NGC CLI. The following command saves the .nemo format model in a folder named `llama-3-8b-instruct-nemo_v1.0` in the current directory. You may choose to specify another path using the `-d` option in the CLI tool.
+
+.. code:: bash
+   
+   ngc registry model download-version "nvidia/nemo/llama-3-8b-instruct-nemo:1.0"
+
+
+``NOTE``: Alternatively, you may choose to download the model from `Hugging Face <https://huggingface.co/meta-llama/Meta-Llama-3-8B-Instruct>`__, and convert it to the `.nemo` format using the Hugging Face to NeMo `Llama checkpoint conversion script <https://docs.nvidia.com/nemo-framework/user-guide/latest/nemotoolkit/ckpt_converters/user_guide.html#community-model-converter-user-guide>`__.  To skip this extra step, the .nemo model has been made available at NGC as mentioned above.
 
-1. Run the container using the following command. It assumes that you have the notebook(s) available in the current working directory. If not, mount the appropriate folder to ``/workspace``.
+2. Run the container using the following command. It assumes that you have the notebook(s) and llama-3-8b-instruct model available in the current directory. If not, mount the appropriate folder to ``/workspace``.
```

Comment:
. Run the container using the following command. It is assumed that you have the notebook(s) and llama-3-8b-instruct model available in the current directory. If not, mount the appropriate folder to ``/workspace``.